contrib/completions/bash/openshift

```diff
@@ -279,442 +279,6 @@ _openshift_options()
     noun_aliases=()
 }
 
-_openshift_start_etcd()
-{
-    last_command="openshift_start_etcd"
-    commands=()
-
-    flags=()
-    two_word_flags=()
-    local_nonpersistent_flags=()
-    flags_with_completion=()
-    flags_completion=()
-
-    flags+=("--config=")
-    flags_with_completion+=("--config")
-    flags_completion+=("__openshift_handle_filename_extension_flag yaml|yml")
-    local_nonpersistent_flags+=("--config=")
-    flags+=("--loglevel=")
-    flags+=("--logspec=")
-
-    must_have_one_flag=()
-    must_have_one_flag+=("--config=")
-    must_have_one_noun=()
-    noun_aliases=()
-}
-
-_openshift_start_master_api()
```

Comment:
didn't expect all of these to disappear...

Reply:
> didn't expect all of these to disappear...

And joy.  Completions are messed up.  The output from the commands is correct (that's what I checked).  Completions are not.

Truth be told, I don't know that I care about completions on these.....